templates/macros/filters/date.html

```diff
@@ -22,9 +22,9 @@
   </ul>
   <div class="date-range-input">
     <label for="min_{{ name }}">From</label>
-    <input type="text" id="min_{{ name }}" name="min_{{ name }}" class="js-min-date" data-inputmask="'alias': 'mm/dd/yyyy'">
+    <input type="text" id="min_{{ name }}" name="min_{{ name }}" class="js-min-date" data-inputmask="'alias': 'mm-dd-yyyy'">
```

Comment:
Yes, but not without making other changes first. We're basically plugging the filter values directly into the URL at the moment, which means we need to use the same date format in the URL and in the filters. I can get around this you think it would be better to use slashes here.